src/main/java/org/opensearch/performanceanalyzer/PerformanceAnalyzerPlugin.java

```diff
@@ -189,27 +195,19 @@ public PerformanceAnalyzerPlugin(final Settings settings, final java.nio.file.Pa
         clusterSettingsManager.addSubscriberForIntSetting(
                 PerformanceAnalyzerClusterSettings.PA_NODE_STATS_SETTING, nodeStatsSettingHandler);
 
-        scheduledMetricCollectorsExecutor.addScheduledMetricCollector(
-                new ThreadPoolMetricsCollector());
         scheduledMetricCollectorsExecutor.addScheduledMetricCollector(
```

Comment:
Can we please extract out these collectors to 2 different methods like collectors and RTF collectors? I will be easy to switch over then.